stacks/api/passport/apikey/apikey.controller.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+  UseGuards
+} from "@nestjs/common";
+import {AuthGuard} from "@nestjs/passport";
+import {Schema} from "@spica-server/core/schema";
+import {ObjectId, OBJECT_ID} from "@spica-server/database";
+import * as uniqid from "uniqid";
+import {ActionGuard} from "../policy/action.guard";
+import {ApiKeyService} from "./apikey.service";
+import {ApiKey} from "./interface";
+
+@Controller("passport/apikey")
+export class ApiKeyController {
+  constructor(private aks: ApiKeyService) {}
+
+  @Get()
+  @UseGuards(AuthGuard(), ActionGuard("passport:apikey:index"))
+  find() {
+    return this.aks.find();
```

Comment:
We need to add filter params to this method like skip and limit for pagination changes.

Reply:
There was no such requirement as far as I know.